CONTRIBUTING.md

```diff
@@ -352,3 +355,30 @@ To set this up, in the Before launch section of any Run/Debug configuration, do
 You only need to do most of these steps once, when you create the
 External Tool. Then you can use it in any run config that spawns the
 agent, to rebuild it first.
+
+## Debugging VS Code
+
+Sometimes, the Agent behaves differently when called from IntelliJ than
+when called from VS Code, and you may need to debug the same code paths
+through the Agent twice. First, when called from the JetBrains extension
+side, and again when called from the VS Code extension side.
+
+To accomplish the latter, you can use VSC to debug itself. It works very
+similarly to how it works in JetBrains. There are predefined run
+configurations for debugging VSC Cody in the file `.vscode/launch.json`
+in `sourcegraph/cody`, such as `Launch VS Code Extension (Desktop)`.
+
+You do not launch VSC run configurations from the command palette.
+Instead, use ctrl-shift-D to open the Run and Debug view, and you can
+see the configuration dropdown at the top.
+
+## Known Issues
+
+- Force-stopping the target often corrupts the indexes, requiring an
```

Comment:
Naive question but what are "the" indexes?

Nit but can we tighten this up... a list with one bullet, with a sub bullet, and workarounds mentioned in the sub bullet and the next paragraph?

Reply:
Fixed.